### PR TITLE
Tech feedback on theguardian.com

### PR DESCRIPTION
**Pull request by johnduffell**
I did some work a while ago to put a tech feedback link in the footer.  It sent a beacon back when people click it.  We usually got about 30 clicks an hour, but it did peak sometimes for some reason.  The idea was to notice which browsers had the big problems and which pages had problems.

I have updated the target to be on theguardian.com rather than beacon.guim, under /info/tech-feedback.
This is so ophan can track things and we'll get the referrer and browse type etc.
There's also a platform PR which also needs a :+1: for the routes.

I'm slightly concerned that ophan won't work unless javascript is working, and since this is a means of knowing when things aren't working for people we really need it to be working.  However hopefully we can get some useful info regardless and maybe iterate on it again.

### CODE REVIEW

**Comment by piuccio** [comment]
ophan might not work, but omniture does, even if the user doesn't have javascript

**Comment by ScottPainterGNM** [comment]
Does this allow people to feedback on issues where they have them? It seems that the app feedback they are getting has been hugely valuable in tracking bugs in certain areas so would be keen we added this to the site if possible?

**Comment by johnduffell** [comment]
@ScottPainterGNM this implementation will just store the referrer, but I might add some click buttons to let them give more data.

**Comment by gklopper** [comment]
I asked Andrea to help with the messaging on this.

Other than that :+1: (but I really want the messaging right first).

File: onward/app/controllers/TechFeedbackController.scala [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import java.net.URLEncoder
+
+import common._
+import model.Cached
+import model.Cached
+import play.api.mvc.{Content => _, _}
+
+object TechFeedbackController extends Controller with Logging {
+
+  def escape(values: Seq[Option[(String, String)]]) = {
```
Comment (janua) [inline onward/app/controllers/TechFeedbackController.scala]: What calls this?

Comment (johnduffell) [inline onward/app/controllers/TechFeedbackController.scala]: nothing, I've actually removed it in my local copy ;)

**Comment by johnduffell** [comment]
closing as I'll do a new one later

**Pull request closed without merging**